.github/workflows/test-install.yaml

```diff
@@ -26,8 +26,27 @@ env:
   HONEYCOMB_DATASET: litmus tests
 
 jobs:
+  build-matrix:
+    runs-on: ubuntu-latest
+    steps:
+      - id: set-matrix
+        run: |
+          # if github event is workflow_dispatch, set matrix parameteres to the user input
+          if [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
+            echo 'version_matrix=["${{ github.event.inputs.version }}"]' >>  "$GITHUB_OUTPUT"
+            echo 'image_matrix=["${{ github.event.inputs.image }}"]' >> "$GITHUB_OUTPUT"
+          else
+            # set matrix parameters to real matrix values
+            echo 'version_matrix=["2019.8.12","2021.7.2"]' >> "$GITHUB_OUTPUT"
```

Comment:
Have you considered the maintenance of the hardcoded releases?

you could so something like this :

https://github.com/puppetlabs/support-tasks/blob/e9303065af9700705920dcc31184ed59ef95b422/.github/workflows/pe_latest_testing.yml#L55

and 

https://github.com/puppetlabs/support-tasks/blob/e9303065af9700705920dcc31184ed59ef95b422/.github/workflows/pe_lts_testing.yml#L60

To suck in the latest version and the current LTS version, so you don't have to modify this when pe releases come out